src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml

```diff
@@ -48,230 +56,88 @@
-    {
-        if (Model.IsFirstRoundGame)
-        {
-            <div class="container-fluid">
-                <div class="row">
-                    <div class="col-md-3">
-                        <strong>@Model.DisplayGameNumber </strong>
-                    </div>
-                    <div class="col-md-7">
-                        <strong>   @Model.HomeTeamName </strong>
-                    </div>
-                </div>
-            </div>
-
-            <div class="row">
-                <div class="col-md-10 text-right">
-                    @Resources.UI.TournamentViews.TeamDayOff
-                </div>
-            </div>
-        }
-        else
-        {
-            <div class="container-fluid">
-                <div class="row">
-                    <div class="col-md-3 game-number">
-                        <strong>@Model.DisplayGameNumber </strong>
-                    </div>
-                    <div class="col-md-7">
                         <div class="row">
-                            <strong>   @Model.HomeTeamName </strong>
+                            <div class="col-md-10">
+                                <strong>  @Model.HomeTeamName </strong>
+                            </div>
+                            <div class="col-md-2">
+                                <strong>   @Model.GameScore.Home </strong>
+                            </div>
                         </div>
                         <div class="row">
-                            <strong>  @Resources.UI.TournamentViews.AwayTeamPlaceholder </strong>
+                            <div class="col-md-10">
+                                <strong>  @Model.AwayTeamName </strong>
+                            </div>
+                            <div class="col-md-2">
+                                <strong>  @Model.GameScore.Away </strong>
+                            </div>
                         </div>
                     </div>
                 </div>
             </div>
+        </div>
 
-            if (Model.GameDate.HasValue)
+        <div class="row">
+            @foreach (var score in Model.SetScores)
             {
-                <div class="row">
-                    <div class="col-md-10 text-right">
-                        @Model.ShortGameDate
+                if (!score.IsEmpty)
+                {
+                    <div class="col-md-2">
+                        @score.Home:@score.Away
                     </div>
-                </div>
+                }
             }
-        }
+        </div>
 
         if (Model.AllowedOperations.IsAllowed(AuthOperations.Games.Edit))
         {
             <div class="row">
                 <div class="col-md-2">
-                    @Html.ActionLink(Resources.UI.TournamentViews.Edit, "EditScheduledGame", new { gameId = Model.Id })
+                    @Html.ActionLink(Resources.UI.TournamentViews.Result, "Edit", "GameResults", new { Id = Model.Id }, null)
```

Comment:
Why this one was changed?

Reply:
![image](https://user-images.githubusercontent.com/11539042/38290356-97dbed08-37e3-11e8-8b12-ffad0b79b5b3.png)

Reply:
It's a little bit lower the page. Which is why I agree with Roman, that GitHub mistakenly shows this as editing, considering the fact that a lot of contitional statements were removed.